rethinkdb/content.md

```diff
@@ -0,0 +1,50 @@
+# What is RethinkDB?
+
+RethinkDB is an open-source, distributed database built to store JSON documents
+and effortlessly scale to multiple machines. It's easy to set up and learn and
+features a simple but powerful query language that supports table joins,
+groupings, aggregations, and functions.
+
+%%LOGO%%
+
+# How to use this image
+
+## start an instance with data mounted in the working directory
+
+The default CMD of the image is `rethinkdb --bind all`, so the RethinkDB daemon
+will bind to all network interfaces available to the container (by default,
+RethinkDB only accepts connections from `localhost`).
+
+```bash
+docker run --name some-rethink -v "$PWD:/data" -d rethinkdb
```

Comment:
For now, would you mind switching to `-v "$(pwd)":/data` so that we're consistent with all the other current docs, and then we can open a new issue/PR to discuss this change across the board?  I'm +1 on the change from a quick bit of research, but I'd rather we stay consistent and change this all at once if we're going to.